runtime/help/options.md

```diff
@@ -197,6 +197,28 @@ Here are the options that you can set:
 
 	default value: `4`
 
+* `numberedtabs`: Display the tab's number beside it in the tab bar.  
+	default value: `on`
+
+* `showclock`: Display a clock in the status line. 
+	(May be off by up to 20 seconds to save on performance. )
+	*Note:* Setting this causes micro to redraw the screen at least 4 times a 
+	minute, instead of just when it receives input. Hence why it is off by default.
+	
+	default value: `off`
+
+* `12hourclock`: Display time in a 12-hour format, rather than the default 24-hour.
+
+	default value: `off`
+	
+* `showseconds`: Display seconds in the status line clock. 
+	(This is a bit more resource intensive as it redraws the screen every second.)
+	
+	default value: `off`
```

Comment:
Maybe have `12hourclock` and `showseconds` say they depend on `showclock`, similar to how `tabmovement` says it depends on `tabstospaces`